### PR TITLE
MIG-1475: Release notes for MTC 1.7.14

### PR DESCRIPTION
**Pull request by anarnold97**
### Issue

* [MIG-1475](https://issues.redhat.com/browse/MIG-1475)

#### Version(s):

* Enterprise 4.11 → branch/enterprise-4.11
* Enterprise 4.12 → branch/enterprise-4.12
* Enterprise 4.13 → branch/enterprise-4.13
* Enterprise 4.14 → branch/enterprise-4.14

#### Issue:

* https://issues.redhat.com/browse/MIG-1475

### Link to docs preview:

* [MTR 1.7.14 release notes](https://66760--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/mtc-release-notes#migration-mtc-release-notes-1-7-14_mtc-release-notes)

QE review:
- [X](https://github.com/openshift/openshift-docs/pull/66760#pullrequestreview-1693775509) -  QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/migration-mtc-release-notes-1-7-14.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-14_{context}"]
+= {mtc-full} 1.7.14 release notes
+
+[id="resolved-issues-1-7-14_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.CVE-2023-39325 CVE-2023-44487 openshift-velero-plugin-container: various flaws
```
Comment (rayfordj) [inline modules/migration-mtc-release-notes-1-7-14.adoc:13]: Consider removing the "openshift-velero..." text as the preceding CVEs affect more than that single image (as noted by the BZ list which follows below).

**Review by rayfordj — CHANGES_REQUESTED** [review]

File: modules/migration-mtc-release-notes-1-7-14.adoc (hunk 1 above)
Comment (anarnold97) [reply modules/migration-mtc-release-notes-1-7-14.adoc]: Yeah I will remove from both headers
I need to tidy up the attributes as well

**Review by rayfordj — APPROVED** [review]
/lgtm

**Review by ShaunaDiaz — COMMENTED** [review]
Complex release notes, great job. Some of these notes could benefit from having long sentences broken up and using active voice.

**Comment by kcarmichael08** [comment]
From a merge review standpoint, everything looks good to me. I was wondering why parentheses were used to set off all the Bugzilla and CVE entries, but I guess it's something that is normally done for this doc set? (It doesn't seem needed in these cases.)

**Comment by anarnold97** [comment]
@kcarmichael08 
```
From a merge review standpoint, everything looks good to me. I was wondering why parentheses were used to set off all the Bugzilla and CVE entries, but I guess it's something that is normally done for this doc set? (It doesn't seem needed in these cases.)
```
I know, right... cos it looks really weird!

**Pull request merged**